Backgrounds and Icons

kowalskiLeon opened the pull request:
Adding icons for the navigation bar menu. I will be changing the .xml files that are being used to .png due to the fact that .png files are easier to be edited.
Also adding backgrounds for the Moeda screen and for the navigation bar.
Background of Moeda was created considering it can't be polluted and it can't obscure the font.
Icons are currently being displayed in black and white.

kowalskiLeon committed:
Backgrounds and Icons

Adding icons for the navigation bar menu. I will be changing the .xml files that are being used to .png due to the fact that .png files are easier to be edited.
Also adding backgrounds for the Moeda screen and for the navigation bar.
Background of Moeda was created considering it can't be polluted and it can't obscure the font.

tabocu reviewed (changes requested):
It wont be approved without the images inserted on layout

kowalskiLeon committed:
Lighter Background

Edited version of the background.
New version has been illuminated in order to avoid disturbance during reading.
